tests/unit/core/test_tracking.js

```diff
@@ -127,3 +127,43 @@ describe('getAction', () => {
     expect(getAction('whatever')).toEqual('invalid');
   });
 });
+
+describe('Do Not Track', () => {
+  it('should respect DNT when enabled', () => {
+    expect(doNotTrackEnabled({
+      _navigator: { doNotTrack: '1' },
+      _window: {},
+    })).toBeTruthy();
```

Comment:
Given the actual code checks for truthiness over `=== true` I thought this was fine, but that's true. I'll change it 😄